justfile

```
@@ -440,7 +451,62 @@ _linkerd-viz-uninit:
 
 
 # TODO linkerd-jaeger-install
-# TODO linkerd-multicluster-install
+
+##
+## linkerd multicluster
+## 
+
+mc-gateway-image := "gcr.io/google_containers/pause:3.2"
```

Where is the source of truth for this image. Can this be read from a values.yaml?

---

I see that this is hardcoded in the template. Ideally we should move this into the values and access it here from there.

At the very least, we should change the name to have a `_` prefix so it's not exported from `--help` etc.
```suggestion
_pause-image := "gcr.io/google_containers/pause:3.2"
```

---

Yeah this is a great idea. I saw that you did something similar in the commit linked further down in the PR. I thought maybe it's best to make it as a separate change (it'd be easier to review). I've opened up https://github.com/linkerd/linkerd2/pull/9566.